frontend/express/views/dashboard.html

```diff
@@ -2501,7 +2504,9 @@ <h4>{{this.name}}</h4>
 		 Countly.userData.set("lastVersion", "<%= countlyVersion %>");
 		 Countly.userData.set("lastEdition", "<%= countlyTypeTrack %>");
 		 Countly.userData.set("isTrial", <%= countlyTrial %>);
-		 Countly.userData.set("cpus", <%= cpus %>);
+         <% if (!offline_mode) { %>
+            Countly.userData.set("cpus", <%= cpus %>);
+         <% } %>
```

Comment:
I think whole SDK usage should be excluded in offline mode, not one random SDK call :)